UniqueFilenameClass/Arguments.cs

```diff
@@ -0,0 +1,29 @@
+﻿using MyConsoleApp.Interfaces;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MyConsoleApp.UniqueFilenameClass
+{
+    public class Arguments : IArguments
+    {
+        public List<ArgumentsDto> ParseArguments(string[] args)
+        {
+            var argumentsList = new List<ArgumentsDto>();
+
+            foreach(string arg in args)
+            {
+                string[] serverAndAppId = arg.Split(',').Select(argValue => argValue.Trim()).ToArray();
```

Comment:
Not quite sure of the value LINQ brings here, and if not really adding value it makes the code a bit harder to read

Reply:
i really need this line to get the server name and app id